docs/current/getting-started.md

```diff
@@ -59,6 +59,16 @@ By default, the only ENVIRONMENT variable set is `VS_NUM=1`.
 
   For example: `VS_NUM=3 vagrant up` will stand up 3 vaquero server VMs. Running `vagrant destroy -f` will only destroy the first instance, you must run `VS_NUM=3 vagrant destroy -f` to clean up all of them. Include *every* ENV var for *every* vagrant command, even things like `vagrant ssh vs-3`.
 
+  **Etcd check:** Once the VM(s) are booted and before running vaquero, ssh into one of your server machines. Ensure etcd is running properly by setting and getting a few keys, as follows:
```

Comment:
can we add the `ETCDCTL_API=3 etcdctl cluster-health`? If you stand up multiple VS machines you might have to wait a bit to get a consensus from the cluster.